### DISCUSSION TITLE
Include javascript in entry

### DISCUSSION THREAD

**Discussion by baer-CH** [bolt/core#2643] [Q&A]
I'm not sure if I'm right here to ask such questions.... ill try
I'd like (or my customer) to include some javascript in a entry's body.
If I safe the entry the "<script>" tag disappears (I think that's  like this for security reasons?).
But... is there a possibility to include javascript?

**Comment by I-Valchev** [comment]
Hey @baer-CH ,

That's definitely a good place to ask a question like that :-) 

You can turn off the sanitisation of the input field that you're referring to, using `sanitise: false`. Here's an example:
https://github.com/bolt/core/blob/4.1/config/bolt/contenttypes.yaml#L373

**Comment by baer-CH** [comment]
Hi I-Valchev
That was Fast! Thank you!
I tried to do so .....
I added `sanitise: false` like this: 
but I have the same problem
```entries:
    name: Entries
    singular_name: Entry
    fields:
        title:
            type: text
            class: large
            group: content
        slug:
            type: slug
            uses: title
        teaser:
            type: html
            type: image
            height: 150px
        body:
            type: html
            height: 300px
            sanitise: false
        image:
            type: image
            group: media
        image_size:
            type: select
            values: [ small, large ]
            label: "Anzeigegrösse"
        video:
            type: video
    relations:
        pages:
          multiple: false
          order: title
          label: Select a page
    taxonomy: [ categories, tags ]
    record_template: entry.twig
    listing_template: listing.twig
    listing_records: 10
    default_status: publish
    sort: -datepublish
    recordsperpage: 10
```

**Reply by baer-CH** [reply]
Thanks again to I-Valchev
just to complete:
the fieldtyp has to be "text" or "textarea", how the example shows, but not explicitly tells.

**Reply by I-Valchev** [reply]
Ah, right! Does it work now? ;-)
